iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/LimitViolationUtils.java

```diff
@@ -94,16 +151,20 @@ public static boolean checkPermanentLimit(ThreeWindingsTransformer transformer,
         return checkPermanentLimit(transformer, side, limitReductionValue, getValueForLimit(transformer.getTerminal(side), type), type);
     }
 
+    public static PermanentLimitCheckResult checkPermanentLimit(ThreeWindingsTransformer transformer, ThreeSides side, LimitType type, LimitsComputer<Identifiable<?>, LoadingLimits> computer) {
+        return checkPermanentLimit(transformer, side, computer, getValueForLimit(transformer.getTerminal(side), type), type);
+    }
+
     public static Overload checkTemporaryLimits(ThreeWindingsTransformer transformer, ThreeSides side, double limitReductionValue, LimitType type) {
         return checkTemporaryLimits(transformer, side, limitReductionValue, getValueForLimit(transformer.getTerminal(side), type), type);
     }
 
-    public static Optional<? extends LoadingLimits> getLimits(Branch<?> branch, TwoSides side, LimitType type) {
```

Comment:
Can you restore the following methods to avoid an unnecessary breaking change?
- `public static Optional<? extends LoadingLimits> getLimits(Branch<?> branch, TwoSides side, LimitType type);`
- `public static Optional<? extends LoadingLimits> getLimits(ThreeWindingsTransformer transformer, ThreeSides side, LimitType type);`

They should be annotated with `@Deprecated(since = "6.4.0")` and their Javadoc should state to use `public static Optional<LoadingLimits> getLoadingLimits(Identifiable<?> identifiable, LimitType limitType, ThreeSides side)` instead.